File: examples/equos-python-integration/src/python_integration/list_faces.py
from python_integration.client import get_client


def list_faces() -> None:
    client = get_client()
    response = client.faces.list(take=50)

    print(f"{len(response.faces)} of {response.total} face(s):")
    for f in response.faces:
        print(f"- {f.id} · {f.identity.value}")


if __name__ == "__main__":
    list_faces()

File: examples/equos-nodejs-integration/src/list-faces.ts
import { client } from "./client.js";

const { total, faces } = await client.faces.listFaces({ take: 50 });

console.log(`${faces.length} of ${total} face(s):`);
for (const f of faces) {
  console.log(`- ${f.id} · ${f.identity}`);
}
